source/agora/test/Network.d

```diff
@@ -86,3 +86,19 @@ unittest
     auto network = makeTestNetwork!Network(NetworkTopology.Simple, 4);
     network.waitUntilConnected(4);
 }
+
+///
+unittest
+{
+    import std.digest.sha; 
+    auto network = makeTestNetwork!Network(NetworkTopology.Simple, 4);
+    network.waitUntilConnected(4);
+
+    assert (network.apis.keys.length == 4);
```

Comment:
```suggestion
    assert(network.apis.keys.length == 4);
```

Reply:
@Geod24,  Move to source/agora/test/GossipProtocol.d
Check Please ~